Resources/Prototypes/_LostParadise/Entities/Mobs/Customization/slime_fox_parts.yml

```
@@ -0,0 +1,11 @@
+- type: marking
+  id: SlimeFoxEars
+  bodyPart: HeadTop
+  markingCategory: HeadTop
+  speciesRestriction: [SlimePerson]
+  sponsorOnly: true # Lost Paradise Sponsors
```

yamllint (1.35.1)

[warning] too few spaces before comment

(comments)